applications/ChimeraApplication/custom_processes/rotate_region_process.cpp

```diff
@@ -0,0 +1,357 @@
+
+#include "rotate_region_process.h"
+
+// Application includes
+
+namespace Kratos
+{
+
+/// Constructor.
+RotateRegionProcess::RotateRegionProcess(ModelPart &rModelPart, Parameters rParameters)
+    : Process(Flags()), mrModelPart(rModelPart), mParameters(rParameters)
```

Comment:
```suggestion
    : Process(), mrModelPart(rModelPart), mParameters(rParameters)
```
I don't think you need to construct the base class of `Process` in here.